src/contracts/GPv2Settlement.sol

```diff
@@ -300,6 +300,23 @@ contract GPv2Settlement {
         filledAmount[uidKey] = currentFilledAmount;
     }
 
+    /// @dev Allows settlment function to make arbitrary contract executions.
+    /// @param interaction contains address and calldata of the contract interaction.
+    function executeInteraction(GPv2Encoding.Interaction memory interaction)
+        internal
+    {
+        // To prevent possible attack on user funds, we explicitly disable
+        // interactions with AllowanceManager contract.
+        require(
+            interaction.target != address(allowanceManager),
+            "GPv2: forbidden interaction"
+        );
+        // solhint-disable-next-line avoid-low-level-calls
+        (bool success, bytes memory response) =
+            (interaction.target).call(interaction.callData);
```

Comment:
Looks like the lint error is back.

Reply:
I'm not sure I follow. Response was reintroduced to bubble up the revert message from the interaction itself.

Reply:
![image](https://user-images.githubusercontent.com/4210206/101513062-0787cb80-397c-11eb-835b-9dd5042d0cf2.png)

Reply:
The `solhint-disable-next-line` is not enough in this case.